TodoList/Src/TodoList.Contracts/Repository/IItemRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using TodoList.Contracts.Models;
+
+namespace TodoList.Contracts.Repository
+{
+    public interface IItemRepository
+    {
+        Task<IEnumerable<Item>> GetAllItemsAsync();
+        Task<Item> GetItemAsync(Guid id);
+        Task<Item> AddItemAsync(Item item);
+        Task<Item> EditItemAsync(Item item);
+        Task DeleteItemAsync(Guid id);
```

Comment:
Consider this:
```suggestion
        Task<Item> DeleteItemAsync(Guid id);
```
You don't have to send the item from controller to the client, but if you want to (in the future), you would have to rewrite the repository.

Reply:
Ok.

Could I make this change in the `ItemRepository.cs`?
```c#
        public async Task<Item> DeleteItemAsync(Guid id)
            => await Task.FromResult<Item>(null);
```

Reply:
fbe8b46